contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -105,7 +117,7 @@ contract BiconomyTokenPaymaster is
         address indexed token,
         uint256 indexed totalCharge,
         address oracleAggregator,
-        uint256 premium,
+        uint32 priceMarkup, // percentage... 1e6 = 100% = 1x
```

Comment:
Update this comment as

// Multiplier value to calculate markup, 1e6 means 1x multiplier = No markup